pkg/infra/teams/team1/init.go

```diff
@@ -0,0 +1,20 @@
+/*******************************************************
+* Copyright (C) 2022 Team 1 @ SOMAS2022
+*
+* This file is part of SOMAS2022.
+*
+* This file or its contents can not be copied and/or used
+* without the express permission of Team 1, SOMAS2022
+*******************************************************/
+
+package team1
+
+import (
+	"infra/game/agent"
+	"infra/game/commons"
+)
+
+var InitAgentMap = map[commons.ID]func() agent.Strategy{
+	"CollaborativeAgent": CreateCollaborativeAgent,
+	"SelfishAgent":       CreateSelfishAgent,
```

Comment:
Our agents will not either be completely collaborative or completely selfish, they will be something in between so don't know if this division makes sense.

Reply:
> This is if we want to have multiple types of agents which would be useful for testing hypotheses and drawing conclusions. This will allows us to do a parameter sweep of the proportion of (selfish agents vs distributive agents). Of course since it is a range of values from 0-1 then we are controlling the distribution of agents that are in the lower or upper half

same here. Still a range of values but from either (0-0.5) or (0.5-1) whos numbers can be controlled by environment variables